Makefile

```diff
@@ -36,6 +36,20 @@ ifneq ($(GIT_TAG),)
 	VERSION_METADATA =
 endif
 
+# Setting container tool
+DOCKER_CMD := $(shell which docker 2> /dev/null)
```

Comment:
We should use `command` instead of `which` similar to https://github.com/konveyor/move2kube/blob/main/scripts/install.sh#L42-L47
as recommended in https://mywiki.wooledge.org/BashFAQ/081

See also: https://unix.stackexchange.com/questions/85249/why-not-use-which-what-to-use-then

Reply:
Indeed! My first implementation was using `command` however the GitHub Action fails because that command is not founded. So this is why I changed to use `switch`command.

You could see in this GitHub Action run: https://github.com/konveyor/move2kube-ui/runs/3741132624?check_suite_focus=true

Maybe I am doing sth wrong to use correctly the `command`.

Reply:
```
$ type command
command is a shell builtin
```
I think this is because Github actions uses a different shell instead of `bash`.
I will fix it in the workflow as shown here: https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#example-4

Reply:
Ok! Reverted my changes to use `command`. Could you check that now the PR is working successfully?

Reply:
This should fix it https://github.com/konveyor/move2kube-ui/pull/99

Reply:
@rmarting can you please change back to what you had initially with `command`?